jormungandr/src/network/client/mod.rs

```diff
@@ -189,11 +190,13 @@ impl Client {
         // the buffered solicitation before polling more events from the
         // block subscription stream.
         let mut client_box = Pin::new(&mut self.client_box);
-        ready!(client_box.as_mut().poll_ready(cx));
+        let logger = &self.logger;
+        ready!(client_box.as_mut().poll_ready(cx))
+            .unwrap_or_else(|e| debug!(logger, "error processing block event"; "reason" => %e));
```

Comment:
In fact, why should this sink error be handled differently than `start_send` below? Use `map_err` and `?` here as well?

Reply:
Yep, I missed this one from the last fix. Sorry.